docs/src/references/mqtt-api.md

```diff
@@ -647,6 +647,64 @@ tedge mqtt pub -r te/device/main///cmd/config_snapshot '{
 }'
 ```
 
+## Signals
+
+Signals are stateless, making them ideal for one-shot requests.
+The topic scheme for signals can be visualized using the diagram below.
+
+<p align="center">
+
+```mermaid
+graph LR
+  te --/--- identifier --/--- cmd
+  subgraph root
+    te
+  end
+
+  subgraph identifier
+    identifier2["&lt;identifier&gt;"]
+  end
+
+  subgraph command
+    direction LR
+    cmd --/--- signal_type["&lt;signal_type&gt;"] --/--- action["&lt;action&gt;"]
+  end
+
+```
+
+</p>
+
+Where the signal segments are describe as follows:
+
+| Segment             | Description                                                                                                                      |
+|---------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| &lt;identifier&gt;  | The [identifier](#group-identifier) (e.g. device/service) associated with the signal.                                            |
+| &lt;signal_type&gt; | Signal type. Each signal can define its own payload schema to allow signals to have parameters related to the signal's function. |
+| &lt;action&gt;      | Action for the signal type. e.g. `check`                                                                                         |
```

Comment:
I'm wondering if two levels: `signal_type` and `action` are necessary or one would have been sufficient. From the examples listed below it makes sense. But I also feel that semantically `config_type/check` could have been modelled as `config_check` as well. But yeah, no harm keeping 2 levels and it might even open up further extension possibilities in future.